Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -244,11 +244,9 @@ def longLineLinter : Linter where run := withSetOptionIn fun stx ↦ do
     -- if the linter reached the end of the file, then we scan the `import` syntax instead
     let stx := ← do
       if stx.isOfKind ``Lean.Parser.Command.eoi then
-        let fname ← getFileName
-        if !(← System.FilePath.pathExists fname) then return default
-        let contents ← IO.FS.readFile fname
+        let contents := (← read).fileMap.source
```

Comment:
I believe this is available:
```suggestion
        let contents := (← getFileMap).source
```

You could also save rebuilding the filemap by creating the input context yourself:
```
let fileMap ← getFileMap
let (impMods, _) ← Parser.parseHeader { input := fileMap.source, fileName := ← getFileName, fileMap := fileMap }
```
(Untested, and I made sure that line was extra long for you.)

Reply:
Kyle, thanks a lot: I accepted both of your suggestions!